layers/meta-resin-allwinner/recipes-kernel/xradio-firmware/xradio-firmware.bb

```diff
@@ -0,0 +1,22 @@
+DESCRIPTION = "Xradio xr819 WiFi firmware"
```

Comment:
I see this recipe is in the thud branch of meta-sunxi. Can you remove it from here?
We will have thud support shortly, it will be best if we do not duplicate it here also

Reply:
I agree with not having duplicates, but the recipe in meta-sunxi is incorrect as of now.
Already created a pr for this, but no one had a look so foar:
https://github.com/linux-sunxi/meta-sunxi/pull/245

Seems to be pretty inactive recently.
For the time beeing I'll remove this from the pr